introduction.tex

```diff
@@ -1,8 +1,18 @@
 \section{Introduction}
 % Convince the reader that Distributed Machine Learning is significant / important / interesting
-% Convince the reader that we shouldn't be completely satisfied with the existing literature on this topic and that our research will address some imiportant limitation or deficiency
+% Convince the reader that we shouldn't be completely satisfied with the existing literature on this topic and that our research will address some important limitation or deficiency
 % Explain our
 	% RESEARCH QUESTION
 	% SUB-RESEARCH QUESTIONS
 	% ALL OF THEM WELL-MOTIVATED
 % Give a hypothesis for them
+
+% Clearly formulated andwell-motivated research question. Main research question split up into sub-questions.
+% Very well-structured document. General presentation of the content (text and figures) is very effective. 
+% Excellent expressed and formulated report. Document has a smooth flow. Spelling and grammatically error free.
+
+With the ever increasing amount of data created every day and the prevalence of Machine Learning (ML) algorithms in analyzing all this data, a lot of research is being done on the application of Distributed Machine Learning approaches. Because of the many different approaches to dealing with so-called Big Data, we would like to give an overview of the current state-of-the-art of Distributed Machine Learning.
```

Comment:
Could be more formal ("With" -> "Due to", "is being done" -> "is occurring", "Because of" -> "To ", "we would like to give" -> "we provide") + final sentence could be clearer on how big data relates to DML.